File-Formats/DocIO/Working-with-Charts/Format-Axis.md

```diff
@@ -0,0 +1,218 @@
+chart.PrimaryValueAxis.Font.FontName = "Calibri";
+chart.PrimaryValueAxis.Font.Bold = true;
+chart.PrimaryValueAxis.Font.Size = 20;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Text Angle Rotation
+
+The following code snippet illustrates how to rotate the text angle for the axis title area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Axis title area text angle rotation.
+chart.PrimaryValueAxis.TitleArea.TextRotationAngle = 90;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Maximum value in Axis
+
+The following code snippet illustrates how to set the maximum value in the axis.
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Maximum value in the axis
+chart.PrimaryValueAxis.MaximumValue = 14.0;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Number format in Axis
+
+The following code snippet illustrates how to set the number format in the axis.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Number format for axis
+chart.PrimaryValueAxis.NumberFormat = "0.0";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Category Label
+
+The following code snippet illustrates how to set the category label.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Sets category labels
+chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 11, 1];
+
+{% endhighlight %}
+{% endtabs %}
+
+## Manage Gridline Visibility
+
+The following code snippet illustrates how to hide or show major and minor gridlines.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Hiding major gridlines
+chart.PrimaryValueAxis.HasMajorGridLines = false;
+
+//Showing minor gridlines
+chart.PrimaryValueAxis.HasMinorGridLines = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Resize the Axis Title Area
+
+The following code snippet illustrates how to resize the axis title area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Manually resizing axis title area using Layout
+chart.PrimaryValueAxis.TitleArea.Layout.Left = 15;
+chart.PrimaryValueAxis.TitleArea.Layout.Top = 20;
+chart.PrimaryCategoryAxis.TitleArea.Layout.Left = 25;
+chart.PrimaryCategoryAxis.TitleArea.Layout.Top = 20;
+
+//Manually resizing axis title area using Manual Layout
+chart.PrimaryValueAxis.TitleArea.Layout.ManualLayout.Left = 0.04;
+chart.PrimaryValueAxis.TitleArea.Layout.ManualLayout.Top = 0.34;
+chart.PrimaryCategoryAxis.TitleArea.Layout.ManualLayout.Left = 0.38;
+chart.PrimaryCategoryAxis.TitleArea.Layout.ManualLayout.Top = 0.95;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Secondary Axis
+
+The following code snippet illustrates how to use secondary axis in chart.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Use Secondary Axis
+chart.Series[1].UsePrimaryAxis = false;
+
+//Set title for secondary value axis
+chart.SecondaryValueAxis.Title = "Temperature,deg.F";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Secondary Value Properties
+
+The following code snippet illustrates how to customize secondary value axis in chart.
```

Comment:
The following code sample illustrates how to customize secondary value axis in chart.